app/src/did/web/method.h

```diff
@@ -48,11 +48,8 @@ namespace scitt::did::web
         const auto& primary_interface =
           node_info->rpc_interfaces.begin()->second;
 
-        // We ignore the hostname part of the bind address, since that is
-        // typically something like 0.0.0.0, and hardcode localhost instead.
-        auto [_host, port] =
-          ccf::split_net_address(primary_interface.bind_address);
-        return fmt::format("https://localhost:{}/did/{}/doc", port, did);
+        return fmt::format(
```

Comment:
I think this deserves a comment noting that `bind_adress` could be 0.0.0.0, but it works fine, given this behaviour is actually a little surprising.

Reply:
Done.